velox/row/tests/UnsafeRowBatchDeserializerTest.cpp

```diff
@@ -689,6 +691,45 @@ TEST_F(
   assertEqualVectors(inputVector, outputVector);
 }
 
+TEST_F(
+    UnsafeRowComplexBatchDeserializerTests,
+    UnsafeRowDeserializationNullRowsTests) {
+  // Single level null RowVector
+  for (auto& batchSize : {1, 5, 10}) {
+    std::vector<std::optional<std::string_view>> serializedVec;
+    const auto& inputVector =
+        createInputRow(batchSize, [](vector_size_t i) { return true; });
+    for (size_t i = 0; i < batchSize; ++i) {
+      serializedVec.push_back(std::nullopt);
+    }
+    VectorPtr outputVector =
+        UnsafeRowDynamicVectorBatchDeserializer::deserializeComplex(
```

Comment:
Can we write this test as a round-trip, i.e. 

- generate input data
- serialize 
- deserialize
- assert that result matches input

Reply:
I thought this test did what you described. Maybe I'm missing something. Could you please help to clarify a bit? Thanks

Reply:
@tanjialiang I'm not seeing 'serialize' step here, rather the test seems to hard-code serialized values.